src/controllers/controllermanager.cpp

```diff
@@ -218,7 +218,7 @@ QList<Controller*> ControllerManager::getControllerList(bool bOutputDevices, boo
     return filteredDeviceList;
 }
 
-QString ControllerManager::getConfiguredPresetFileForDevice(const QString& name) {
+QString ControllerManager::getConfiguredMappingFileForDevice(const QString& name) {
     return m_pConfig->getValueString(ConfigKey("[ControllerPreset]", sanitizeDeviceName(name)));
```

Comment:
Unfortunately, this string is duplicated. I suggest to extract into into a local constant and move the comment there.